### ISSUE TITLE
A11y_.NETCore_ASP.NETCore_Login_Usability: Placeholder text for email and password field is not visible in normal mode.

### ISSUE THREAD

**Issue by kupatkar99** [dotnet/aspnetcore#51135]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

### Github Tags:
#A11yUsable; #A11ySev4; #A11yTCS; #DesktopWeb; #SH_.NETCore_Web_Oct2023; #.NET Core; #Win11; #ChromiumEdge; 

### Environment Details:
App name: .NET Core
URL: http://localhost:5045/Account/Login
Chromium Edge: Version 119.0.2132.0 (Official build) dev (64-bit)

### Actual Result:
Placeholder text for email and password field is not visible in normal mode.

### User Impact:
User will face difficulty to know the format for input field as the place holder is not visible in normal mode.

### Expected Behavior

 Placeholder text for email and password field should be visible in normal mode.

### Steps To Reproduce

1. Open command prompt and enter command "mkdir foldername" e.g. "mkdir BlazorWasm".
2. Move to the created directory using command "cd foldername".
3. Run command "dotnet new blazorwasm" and hit ENTER key.
4.Run command "dotnet run".
5. Copy the First URL link from Command prompt and paste it in URL section of Browser and hit ENTER key.
6. TAB to "Login" control from left navigating and press ENTER.
7. "Login" screen will get open.
8. TAB to "Use A Local Account to Login" section.
9. Observe that Placeholder text for email and password field is visible in normal mode or not
<img width="956" alt="A11y_ NETCore_ASP NETCore_Login_Usability" src="https://github.com/dotnet/aspnetcore/assets/98530069/a9a3d88a-f37f-43c3-a608-6f7f7da6a65d">

**Comment by mkArtakMSFT** [comment]
@kupatkar99 can you please clarify what the `Normal` mode is and also a screenshot where you see the text is not shown?
In your current screenshot, the `Email` and `Password` texts are obviously visible.

**Comment by kupatkar99** [comment]
@mkArtakMSFT the Placeholder text is visible in high contrast mode but similarly it's placeholder text is not visible in normal 100% zoom mode. please refer below attach screenshot.
<img width="911" alt="image" src="https://github.com/dotnet/aspnetcore/assets/98530069/3c71a30b-f632-4943-ab21-4622ac971698">

**Comment by javiercn** [comment]
Seems this is a bug in bootstrap. 

This rule seems to be the culprit, but even in this case, it doesn't really work, as providing a color makes the placeholder overlap with the label text.

![image](https://github.com/user-attachments/assets/f8ccacdc-c1e2-4ece-ab9b-c9c01d4a37ba)

These two rules on CSS address the situation:

```css
.form-floating > .form-control-plaintext::placeholder, .form-floating > .form-control::placeholder {
    color: var(--bs-secondary-color);
    text-align: end;
}

.form-floating > .form-control-plaintext:focus::placeholder, .form-floating > .form-control:focus::placeholder {
    text-align: start;
}
```

https://github.com/user-attachments/assets/26505d00-f406-4892-9022-a9c7b0b04e23

**Comment by kupatkar99** [comment]
#A11yFixed; #Closed;
